db/data_migration/20150317165803_republish_case_studies_to_render_on_government_frontend.rb

```diff
@@ -0,0 +1 @@
+DataHygiene::PublishingApiRepublisher.new(CaseStudy.where(:state => Edition::PUBLICLY_VISIBLE_STATES)).perform
```

Comment:
sorry, i was familiarising myself with the intention behind pushing "unpublishings" to content-store, looking at https://github.com/alphagov/whitehall/pull/1908 and https://github.com/alphagov/whitehall/pull/1891. if i understand correctly, we need to re-publish case studies in a publicly visible state, and re-publish the unpublishings of case studies that have been unpublished. whereas, your example republishes case studies that have been unpublished.

there's coming soon as well, but i don't understand that completely to know whether we need to consider those in the migration.